core/node/node_framework/src/implementations/layers/pk_signing_eth_client.rs

```diff
@@ -94,120 +94,198 @@ impl WiringLayer for PKSigningEthClientLayer {
         let signing_client_for_gateway;
         let mut signing_client_for_blobs = None;
 
-        match self.client_type {
-            SigningEthClientType::PKSigningEthClient => {
-                let private_key = self.wallets.operator.private_key();
-
-                let sc = PKSigningClient::new_raw(
-                    private_key.clone(),
-                    self.contracts_config.diamond_proxy_addr,
-                    gas_adjuster_config.default_priority_fee_per_gas,
-                    l1_chain_id,
-                    query_client.clone(),
-                );
-                signing_client = BoundEthInterfaceResource(Box::new(sc));
-
-                signing_client_for_blobs = self.wallets.blob_operator.map(|blob_operator| {
-                    let private_key = blob_operator.private_key();
-                    let signing_client_for_blobs = PKSigningClient::new_raw(
-                        private_key.clone(),
+        // if the wallet.yaml has gkms key setup for operator, will use it instead of using the gkms settings in env
```

Comment:
it seems if `gkms_mode: true`  then  `gkms_key_name` is required.

In that case maybe no need to consider the case gkms settings is set in env because it will be always required in the wallet.yaml ?

Reply:
So we decide to run the gkms signer using wallet.yaml? and drop the previous logic to run it with `env`?

Reply:
gotcha, 
okay let keep for now until we get totally rid of the `env` . Those became deprecated anyways. I am not sure at which extend it supports the new configs

Reply:
Removed the logic to load the gkms key name from the env. And using wallet setup instead.